ci/tests/vmachine/vmachine_test.py

```diff
@@ -135,103 +136,72 @@ def check_scrubbing_test():
-            if err:
-                GeneralVMachine.logger.error("Error while stopping vmachine: {0}".format(err))
+        vdisk = VDiskList.get_by_devicename_and_vpool('/' + disk_name + '.raw', vpool)
 
-        vds = GeneralVDisk.get_vdisks()
-        disk_backend_data = {}
-        for disk in vds:
-            # saving disk 'stored' info / the only attribute that is lowered after scrubbing
-            disk_backend_data[disk.guid] = disk.statistics['stored']
+        # saving disk 'stored' info / the only attribute that is lowered after scrubbing
+        disk_backend_data = vdisk.statistics['stored']
 
         # deleting middle snapshots
-        for disk in vds:
-            for snapshot in disk.snapshots[1:-1]:
-                VDiskController.delete_snapshot(disk.guid, snapshot['guid'])
+        for snapshot in vdisk.snapshots[1:-1]:
+            VDiskController.delete_snapshot(vdisk.guid, snapshot['guid'])
 
         # starting scrubber
         ScheduledTaskController.gather_scrub_work()
         # waiting for model to catch up
-        time.sleep(120)
-        for disk in vds:
-            disk.invalidate_dynamics(['statistics'])
-        # checking result of scrub work
-        vds = GeneralVDisk.get_vdisks()
-        for disk in vds:
-            if disk.statistics['stored'] >= disk_backend_data[disk.guid]:
-                issues_found += "No scrub work was applied to {0} disk.\nOld stored data:{1}\nNew stored data:{2}\n".format(disk.name,
-                                                                                                                            disk_backend_data[disk.guid],
-                                                                                                                            disk.statistics['stored'])
-        # cleanup
-        # removing vmachines
-        for vm_number in range(nr_of_disks):
-            vmachine_name = "machine-{0}".format(vm_number)
-            GeneralVMachine.logger.info("Removing vmachine {0}".format(vmachine_name))
-            out, err, _ = General.execute_command('virsh destroy {0}'.format(vmachine_name))
-            GeneralVMachine.logger.info("Removing vmachine {0} - out:".format(out))
-            GeneralVMachine.logger.info("Removing vmachine {0} - err:".format(err))
-            out, err, _ = General.execute_command('virsh undefine {0}'.format(vmachine_name))
-            GeneralVMachine.logger.info("Removing vmachine {0} - out:".format(out))
-            GeneralVMachine.logger.info("Removing vmachine {0} - err:".format(err))
-
+        counter = initial_counter
+        while counter > 0:
+            time.sleep(step)
+            vdisk.invalidate_dynamics(['statistics'])
+            # checking result of scrub work
+            if vdisk.statistics['stored'] < disk_backend_data:
+                GeneralVMachine.logger.info("It took {0} seconds for the value to change from {1} to {2}\n".format(300 - counter,
```

Comment:
This will report a longer time than in reality. More correct would be ```(initial_counter - counter) * step``` since every step takes 5 seconds.